src/Plugin_Command.php

```diff
@@ -645,6 +647,17 @@ protected function get_item_list() {
 				'title' => $details['Name'],
 				'description' => wordwrap( $details['Description'] ),
 			);
+
+			if ( null === $update_info ) {
+
+				// Get all plugin info that don't have an update.
+				$plugin_update_info = ( isset( $all_update_info->no_update[$file] ) ) ? $all_update_info->no_update[$file] : null;
```

Comment:
Superfluous parentheses:
```suggestion
				$plugin_update_info = isset( $all_update_info->no_update[$file] ) ? $all_update_info->no_update[$file] : null;
```